ctapipe/visualization/tests/test_mpl.py

```diff
@@ -103,7 +111,18 @@ def test_array_display():
         name="TestSubarray", tel_positions=tel_pos, tel_descriptions=tels
     )
 
-    ad = ArrayDisplay(sub)
+    # Create a fake event containing telescope-wise information about
+    # the image directions projected on the ground
+    ArrayEventContainer.dl1 = DL1Container()
```

Comment:
You assign here to the class! you have to create an instance. Like this:

```
event = ArrayEventContainer()
```

and then use it.

You can also directly pass all the fields in the constructor.

Reply:
fixed